libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_IP.c

```diff
@@ -1608,6 +1614,16 @@ eFrameProcessingResult_t eReturn = eProcessBuffer;
 	}
 	#else
 	{
+
+		if (eReturn == eProcessBuffer )
+		{
+			if( xCheckSizeFields( ( uint8_t * )( pxNetworkBuffer->pucEthernetBuffer ), pxNetworkBuffer->xDataLength ) != pdPASS )
+			{
+				/* Some of the lengths checks was not successful. */
```

Comment:
grammar: _Some of the length checks were not successful._

Reply:
corrected, thanks